src/app/games/lasca/LascaMove.ts

```diff
@@ -0,0 +1,131 @@
+import { Coord, CoordFailure } from 'src/app/jscaip/Coord';
+import { Move } from 'src/app/jscaip/Move';
+import { ArrayUtils } from 'src/app/utils/ArrayUtils';
+import { assert } from 'src/app/utils/assert';
+import { MoveEncoder } from 'src/app/utils/Encoder';
+import { Localized } from 'src/app/utils/LocaleUtils';
+import { MGPFallible } from 'src/app/utils/MGPFallible';
+import { MGPOptional } from 'src/app/utils/MGPOptional';
+import { MGPSet } from 'src/app/utils/MGPSet';
+import { JSONObject, JSONValue, JSONValueWithoutArray, Utils } from 'src/app/utils/utils';
+
+export class LascaMoveFailure {
+    public static readonly CAPTURE_STEPS_MUST_BE_DOUBLE_DIAGONAL: Localized = () => $localize`Capture must be double diagonal steps`;
+    public static readonly MOVE_STEPS_MUST_BE_SINGLE_DIAGONAL: Localized = () => $localize`Move must be single diagonal steps`;
+    public static readonly CANNOT_CAPTURE_TWICE_THE_SAME_COORD: Localized = () => $localize`You cannot jump over the same coord several time`;
```

Comment:
coord is not a word
coordinate is not a word that should be visible to the user
you don't capture a "coord"